hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/api/impl/TestAHSv2ClientImpl.java

```diff
@@ -228,17 +304,17 @@ private static TimelineEntity createContainerEntity(ContainerId containerId) {
     entityInfo.put(ContainerMetricsConstants.ALLOCATED_MEMORY_INFO, 1024);
     entityInfo.put(ContainerMetricsConstants.ALLOCATED_VCORE_INFO, 8);
     entityInfo.put(ContainerMetricsConstants.ALLOCATED_HOST_INFO,
-        "test host");
+            "test host");
     entityInfo.put(ContainerMetricsConstants.ALLOCATED_PORT_INFO, 100);
     entityInfo
-        .put(ContainerMetricsConstants.ALLOCATED_PRIORITY_INFO, -1);
+            .put(ContainerMetricsConstants.ALLOCATED_PRIORITY_INFO, -1);
     entityInfo.put(ContainerMetricsConstants
-        .ALLOCATED_HOST_HTTP_ADDRESS_INFO, "http://test:1234");
+            .ALLOCATED_HOST_HTTP_ADDRESS_INFO, "http://test:1234");
     entityInfo.put(ContainerMetricsConstants.DIAGNOSTICS_INFO,
-        "test diagnostics info");
+            "test diagnostics info");
     entityInfo.put(ContainerMetricsConstants.EXIT_STATUS_INFO, -1);
     entityInfo.put(ContainerMetricsConstants.STATE_INFO,
-        ContainerState.COMPLETE.toString());
+            ContainerState.COMPLETE.toString());
```

Comment:
These seem to be unrelated, for the sake of easier backports/potential conflicts please remove them.